app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,46 @@
+class IngredientsController < ApplicationController
```

Comment:
Can you remove the commented code from this file? The main branch should not have excess code.

Reply:
done